dll/win32/browseui/lang/bg-BG.rc

```diff
@@ -300,6 +300,7 @@ STRINGTABLE
 BEGIN
     IDS_ADDRESSMENUTEXT "&Address"
     IDS_HISTORYTEXT "&Дневник\tCtrl+H"
+    IDS_CLOSE "&Close\tAlt+F4"
```

Comment:
Is it possible to extract that string programmatically at runtime from a real system-menu, instead of adding a duplicate of it into the browseui resources?

Reply:
Possible. With no check:
```c
HMENU hSysMenu = ::GetSystemMenu(hWnd, FALSE);
MENUITEMINFOW mii = { sizeof(mii), MIIM_TYPE };
WCHAR szText[64];
mii.dwTypeData = szText;
mii.cch = _countof(szText);
GetMenuItemInfoW(hSysMenu, &mii);
... // Use szText
```

Reply:
To me that looks superior, as it is shorter and would improve the locality of this menus implementation, which requires a lot of "manual work" anyway in the code. Would also reduce translation-follow-up-commits. Do you agree?

Reply:
Fixed.

Reply:
Thanks. Looks much better already.
Introduction of define CMDID_CLOSE is inevitable?

Reply:
![still-working](https://user-images.githubusercontent.com/2107452/218076022-89f86b14-10fd-4885-ad3e-429d77901589.png)
Confirmed still working.